packages/graphile-build-pg/src/plugins/introspectionQuery.js

```diff
@@ -69,15 +70,15 @@ with
       pro.proisagg = false and
       pro.proiswindow = false and
```

Comment:
According to https://github.com/graphile/postgraphile/issues/796 these two lines causes PostGraphile to fail to start on PG11; it would be awesome if you could fix this in this PR also, but no worries if not - I can add it after 👍

According to the PG11 docs there's a new attribute `prokind`:

> `f` for a normal function, `p` for a procedure, `a` for an aggregate function, or `w` for a window function

Seems `prokind` did not exist before PG11, so it'd be something like:

```
${serverVersionNum >= 110000 ? "pro.prokind = 'f'" : 'pro.proisagg = false and pro.proiswindow = false'} and
```

I'm not sure if that's all that's required to support PG11.

---

To run PG11 without installing you can use Docker:

```
docker run -it --rm -p 127.0.0.1:54321:5432 postgres:11
```

To connect, you can do:

```
psql postgres://postgres@localhost:54321/postgres
```

For the tests

```
export TEST_DATABASE_URL="postgres://postgres@localhost:54321/postgres"
```

Reply:
Tests now pass with PG11 :tada: